Prepare general docs for GA release

Andriciuc opened the pull request:
Updated the introduction with the proper extension name, updated the intro to reflect this. Removed important note about not meant for production and added the No upgrade warning from previous versions (like RC2) to GA. Updates:

* removed the block announcement for RC2 at the top of the HTML page in the intro
* Added the warning note before installation begins too.
* Updated site name to full name.

Andriciuc committed:
updated intro and site name

Updated the introduction with the proper extension name, updated the intro to reflect this. Removed important note about not meant for production and added the No upgrade warning from previous versions (like RC2) to GA.

Added the warning note before installation begins too.

Updated site name to full name.

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs

Andriciuc committed:
Update main.html

removed the block announcement for RC2 at the top of the HTML page in the intro

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs

dutow commented on contrib/pg_tde/documentation/docs/install.md:
```diff
@@ -1,8 +1,8 @@
-    There is no safe upgrade path from the Release Candidate 2 (RC2) to the General Availability (GA) version of `pg_tde`.  
-    We recommend starting with a **clean installation** for GA deployments. Avoid using RC environments in production. -->
+!!! warning "No upgrade path from RC to GA"
+    There is no safe upgrade path from the previous versions (such as Release Candidate 2) to the General Availability (GA) version of `pg_tde`.  
```
`Candidate 2) ` should be `Candidate 2`

Andriciuc replied:
resolved, removed () and replaced with , as it feels cleaner

Andriciuc committed:
updated the "No upgrade path from RC to GA" warning

Removed parenthesis and replaced with , as it looks cleaner.

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs

Andriciuc committed:
updated the "No upgrade path from RC to GA" warning

Removed parenthesis and replaced with , as it looks cleaner.

Andriciuc committed:
Merge branch 'preparing-ga-release-docs' of https://github.com/percona/postgres into preparing-ga-release-docs

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs

Andriciuc committed:
improved content using feedback updates

removed permissions management from architecture and functions
fixed port descriptions in multi-tenant and kmip server
removed block announcement
updated faq with removal of some unneeded future references
updated limitations
fixed pdf cover page release to 1.0

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs

nastena1606 commented on contrib/pg_tde/documentation/docs/index.md:
```diff
@@ -1,9 +1,10 @@
+# Percona Transparent Data Encryption for PostgreSQL documentation
 
-`pg_tde` is the open source, community driven and futureproof PostgreSQL extension that provides Transparent Data Encryption (TDE) to protect data at rest. `pg_tde` ensures that the data stored on disk is encrypted, and that no one can read it without the proper encryption keys, even if they gain access to the physical storage media.
+Percona Transparent Data Encryption for PostgreSQL (`pg_tde`) is an open-source, community driven and futureproof PostgreSQL extension that provides Transparent Data Encryption (TDE) to protect data at rest. `pg_tde` ensures that the data stored on disk is encrypted, and that no one can read it without the proper encryption keys, even if they gain access to the physical storage media.
```
We don't hyphernate "open-source" 

> **Open source**

Open source is never capitalized, except when in the title case or starting a sentence. **Open source is** **never** **hyphenated**. We do not offer open-source solutions; we offer open source solutions.
Source: https://www.notion.so/percona/Percona-Editorial-Style-Guide-905adcfe778044aa9df6543c038e83d7?source=copy_link

Andriciuc replied:
Fixed! Thank you

Andriciuc committed:
Update index.md

open-source to open source

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs

Andriciuc committed:
Merge branch 'release-17.5.2' into preparing-ga-release-docs
